integration-tests/tests/basic-happy-path.spec.ts

```diff
@@ -41,6 +41,9 @@ describe('Basic Happy Path', () => {
       'Build Running',
       'Default',
     );
+    Applications.goToIntegrationTestsTab().hasIntegrationTest(
```

Comment:
would prefer new test `it()` for it. can we create new test here for Integration test validation?